src/Entities/ArrowEntity.cpp

```diff
@@ -21,6 +21,11 @@ cArrowEntity::cArrowEntity(cEntity * a_Creator, double a_X, double a_Y, double a
 {
 	SetSpeed(a_Speed);
 	SetMass(0.1);
+<<<<<<< HEAD
+	SetGravity(-20.0f);
+	SetAirDrag(0.2);
+=======
+>>>>>>> parent of 5d9eb71... Gave entities more vanilla-like gravity and created 'air drag' property
```

Comment:
Ah yes, don't know how those even got there.  :(  That's why the build failed.  Time to spam more commits.